packages/@webex/internal-plugin-dss/src/dss.ts

```diff
@@ -355,7 +370,7 @@ const DSS = WebexPlugin.extend({
    * @returns {Promise} Resolves with an array of entities found
```

Comment:
timeout applied for all single and batch requests. I updated the JSDocs

Reply:
I guess we have 3 options:
1. don't apply timeout for batched requests at all
2. apply it to the first item in the batcher
3. apply it to the last item in the batcher
I don't mind which one we go for, current implementation is like point 2 and I guess it's fine as long as this is explained in the jsdocs

Reply:
@marcin-bazyl correct me if I am wrong, but the point with batch request is collect all the requests in a specific timeframe and do only one call to the server. Since the the server call falls back to the `_request` function from `dss.ts` the whole batch will fail

Reply:
@maxinteger the point is that each request in a batch is made (potentially) with a different timeout, but only one timeout is applied to the batch, so which one?

Reply:
@lismith2-cisco We store the timeout in the batcher instance, which is created for each resources only once. So the first call on `_batchedLookup` will determine the timeout for all subsequent request on the same resource.

If we drop the batcher instance after the queued items finished then we can set a new timeout for the next batch

Reply:
Sorry, I was incorrect, yes it's per batcher not batch. I don't think dropping the batcher after each batch would be better, and I think it would be extremely hard to do. To apply timeouts properly, create a new batcher based on resource/timeout combination, instead of just resource. Or don't make timeouts configurable for batched requests, and document it. Or don't make timeouts configurable at all (which I would vote for tbh), the global config can be set by the app, and I don't see a use case for each request choosing a timeout.

Reply:
@lismith2-cisco I agree, I dropped the timeout parameter and it always use the default timeout